src/ucs/stats/stats.h

```diff
@@ -74,23 +75,26 @@ size_t ucs_stats_aggregate(
         size_t aggregate_sum_counters_size,
         const ucs_stats_aggrgt_sum_clsid_to_name_db_t **clsid_to_name_db);
 
+
 /**
  * A UCX statistics API to get the names of counters on the last call to
- * ucs_stats_aggregate_sum()
+ * ucs_stats_aggregate(). 
+ *
+ * Note, that ucs_stats_aggregate() must be called before calling this 
+ * function.
  *
  * @param [out] names_db The aggregate-sum counters names database
  * @param [out] size     number of counters in database
  *
  * Complexity: O(1)
  *
- * Returns: None.
+ * @return: None.
```

Comment:
Sorry, no need return tag for void functions

Reply:
Fixed.